test/Cpp/src/hello_bodyless_world.cc

```diff
@@ -0,0 +1,5 @@
+#include "HelloBodylessWorld/HelloBodylessWorld.hh"
+
+void HelloBodylessWorld::Inner::hello_body([[maybe_unused]] const reactor::StartupTrigger& startup) {
```

Comment:
so its always reaction_name + _body ? is this _body necessary ?

Reply:
I don't think it is required, actually.  However, there will be C++ compile errors if a reactor uses e.g. a state variable and a reaction with the same name. We  don't have proper name conflict handling.

Reply:
Opened https://github.com/lf-lang/lingua-franca/issues/1973